src/shared/Notifications/Notifications.js

```diff
@@ -2,66 +2,120 @@ import React, { Component } from 'react';
 import cx from 'classnames';
 import PropTypes from 'prop-types';
 
-import { NOTIFICATION_TYPES } from '../../constants';
-
 import './Notifications.scss';
 
 export const CN = 'notifications';
 
+const notificationIcons = {
+  info: {
+    type: 'info',
+    icon: 'info icon'
+  },
+  success: {
+    type: 'success',
+    icon: 'check circle '
+  },
+  warning: {
+    type: 'warning',
+    icon: 'exclamation triangle'
+  },
+  error: {
+    type: 'error',
+    icon: 'x icon'
+  },
+  change: {
+    type: 'change',
+    icon: 'pencil alternate'
+  },
+  custom: {
+    type: 'custom',
+    icon: 'wrench'
+  }
+};
+
 class Notifications extends Component {
   constructor(props) {
     super(props);
 
-    this.hideMessage = this.hideMessage.bind(this);
+    this.hideNotification = this.hideNotification.bind(this);
+    this.renderNotifications = this.renderNotifications.bind(this);
+    this.renderIcon = this.renderIcon.bind(this);
   }
 
-  hideMessage() {
-    const { isHideNotification } = this.props;
-    isHideNotification();
+  hideNotification(id) {
+    const { removeItemNotification } = this.props;
+    removeItemNotification(id);
   }
 
-  render() {
-    const {
+  renderIcon(type) {
+    switch (type) {
+      case notificationIcons.info.type:
+        return notificationIcons.info.icon;
+      case notificationIcons.success.type:
+        return notificationIcons.success.icon;
+      case notificationIcons.warning.type:
+        return notificationIcons.warning.icon;
+      case notificationIcons.error.type:
+        return notificationIcons.error.icon;
+      case notificationIcons.change.type:
+        return notificationIcons.change.icon;
+      case notificationIcons.custom.type:
+        return notificationIcons.custom.icon;
+      default: return null;
+    }
+  }
+
+
+  renderNotifications() {
+    const { notifications } = this.props;
+    return notifications.map(({
+      id,
       type,
       message,
-      title,
-      isShowNotification
-    } = this.props;
-
-    return isShowNotification && (
-      <div
-        className={cx(CN, type)}
-        color={type}
-        onClick={this.hideMessage}
-      >
-        <div onClick={this.onNotificationClick}>
-          {title && <h4 className="title">{title}</h4>}
-          <div className={`${CN}__notification-message-box`} role="alert">
-            <div className={`${CN}__message`}>{message}</div>
+      title
+    }) => {
+      const iconName = this.renderIcon(type);
+      return (
+        <div
+          className={cx(CN, type)}
+          id={id}
+          key={id}
+          color={type}
+          onClick={() => this.hideNotification(id)}
+        >
+          <div className={`${CN}__ntf-box`}>
```

Comment:
In some style names you use 'ntf', and in some: 'notification'. I believe it would be better to use only one naming approach (maybe second one, because it is more clear).

Reply:
done)